amundsen_application/static/js/ducks/user/api/tests/index.spec.ts

```diff
@@ -0,0 +1,170 @@
+import axios, { AxiosResponse } from 'axios';
+
+import globalState from 'fixtures/globalState';
+
+import { LoggedInUser, PeopleUser, Resource } from 'interfaces';
+
+import {
+  loggedInUser, userById, userOwn, userRead,
+  LoggedInUserAPI, UserAPI, UserOwnAPI, UserReadAPI
```

Comment:
Based on recent naming changes, I think these become less clear.
`UserAPI` is the response type, `userById` is the API function, and `getUser` is the connected dispatch function. On top of this we have the redux action types such as `GetUserRequest` and `GetUserResponse`. 


Maybe something like the following would work better
```
// Action Type
GetUser

// Actions
GetUserRequest
GetUserResponse

// Dispatch function
getUser

// API function
userById -> getUserApi

// API response
UserAPI -> GetUserApiResponse
```

Reply:
We could also use the `import * as` syntax to for grouping and just call everything the same thing.
```
import * as UserAPI from '../v0';
import * as User from '../../reducer'

UserApi.getUser(...);
User.getUser
```

Reply:
**Re: Naming**
I don't intend to do any renaming in this PRs, only what is necessary and affected the unit tests. However I'm not opposed to taking a pass to have better and more consistent names -- it seems we're good on the actions and dispatch methods but things start feeling like "this could be better" when it comes to what's in those `api` directories, correct? I would very much like to clean those up.

For the naming patterns proposed, let me know how you feel about it when not looking a short simple method like `getUser`. 😸 I'm fine to make those changes in a renaming-only PR as long as we're sure we won't want to change it later solely because of long names, or get tempted to use abbreviations instead of being explicit.
Examples:
`updateTableDescriptionApi`
`UpdateTableDescriptionApiResponse`
My vote is for consistency and clarity so I'm fine with `SuperLongNameApiResponse` if that's what it has to be.

Reply:
**Re: Imports**
I'm a fan of the `import * as blah` to get rid of the crazy imports at the beginning of each test and having to add the new method each time. Only took until the monster that is `ducks/tableMetadata` to realize it. Some concerns:
1. The aliases will have to be more specific because naming conflicts, e.g. `import * as User from '../../reducer'`. Not sure if something like `SomethingReducer` is preferable, makes the imports nicer but we have to call `SomethingReducer.getSomethingFailure()`
2. Do you feel the same about the imports from '..types'?

Reply:
I mentioned the names because we did change `getUserById` -> `userById` (API) and `getUserById` -> `getUser`(Reducer). We should 1) have some consistency about the `ById` part and 2) make these more clear as to what they are. I'm fine with revisiting in a separate PR to cleanup names.

For import formats `User` may be a bad example since there is a `User` interface already (or is that `PeopleUser` now?). In general I know you shouldn't use the * import pattern on large packages, but it should be okay `import *` from our own source files. It's not going to increase our bundle size. I don't have a concrete idea on when to use `import *` and when to use named imports, but we could employ it on a per-need basis in test files for now.

Reply:
For my previous comments what I'm getting at is purely whether or not you stand by the naming pattern proposals even if it results in long names. Seems like we're good and I'll use those patterns in the cleanup PR.

Let me know if you have any other concerns on this PR.